addons/payment/views/payment_provider_views.xml

```diff
@@ -220,8 +221,15 @@
         <field name="view_mode">kanban,list,form</field>
         <field name="help" type="html">
             <p class="o_view_nocontent_smiling_face">
-                Create a new payment provider
+                No provider available
             </p>
+            <p class="fw-bold">To add a provider in a company A</p>
+            <ol class="text-start d-inline-block">
+                <li>Enable main company and company A</li>
+                <li>Select an existing provider or install one</li>
+                <li>Duplicate this provider</li>
+                <li>Set company A as company of this provider</li>
+            </ol>
```

Comment:
This phrasing sounds approximate IMHO. Could you change it to this?
![image](https://github.com/user-attachments/assets/2442e9ca-ab30-49ba-87dc-e044648927fc)